java/code/webapp/WEB-INF/decorators/layout_head.jsp

```diff
@@ -37,14 +39,28 @@
     <link rel="stylesheet" href="/javascript/select2/select2-bootstrap.css?cb=${cb_version}" />
 
     <!-- import spacewalk styles -->
-    <rhn:require acl="is(development_environment)">
-      <link rel="stylesheet/less" type="text/css" href="/css/spacewalk.less" />
-      <script>less = { env: 'development' };</script>
-      <script src="/javascript/less.js"></script>
-    </rhn:require>
-    <rhn:require acl="not is(development_environment)">
-      <link rel="stylesheet" href="/css/spacewalk.css?cb=${cb_version}" />
-    </rhn:require>
+    <c:choose>
+        <c:when test='${Config.get().getString("product_name").compareToIgnoreCase("Uyuni") == 0}'>
```

Comment:
The test == 0 means "true" ? Are you sure?

Reply:
Well, I have to admit that I just copy-pasted, but giving it a manual try it worked. Anyway, I took it from [here](https://github.com/uyuni-project/uyuni/blob/master/java/code/webapp/WEB-INF/includes/leftnav.jsp#L8) and AFAIK it is working as expected in the current setup. Am I overlooking something?
I will log the value of the test anyway to double check if this is the proper way of testing it, or there is a better/more robust check function we can use.
Thanks for the heads up

Reply:
Anyway, based on your point that was good enough to raise awareness the check was not really good shaped, I moved to the better method it already exists in `ConfigDefaults.java`. This looks much better now :slightly_smiling_face:

Reply:
Hmm, we call a "compare" function. They may give -1, 0, 1 for lower, equal, greater.
That makes sense